docs/resources/ocean_aks_2.0.md

```diff
@@ -0,0 +1,244 @@
+    * `autoscale_is_enabled` - (Optional, Default: `true`) Enable the Ocean Kubernetes Autoscaler.
+    * `autoscale_down` - (Optional) Auto Scaling scale down operations.
+        * `max_scale_down_percentage` - (Optional) The maximum percentage allowed to scale down in a single scaling action.
+    * `resource_limits` - (Optional) Optionally set upper and lower bounds on the resource usage of the cluster.
+        * `max_vcpu` - (Optional) The maximum cpu in vCpu units that can be allocated to the cluster.
+        * `max_memory_gib` - (Optional) The maximum memory in GiB units that can be allocated to the cluster.
+    * `autoscale_headroom` - (Optional) Spare resource capacity management enabling fast assignment of pods without waiting for new resources to launch.
+        * `automatic` - (Optional) Automatic headroom configuration.
+            * `percentage` - (Optional) Optionally set a number between 0-100 to control the percentage of total cluster resources dedicated to headroom.
+* `controller_cluster_id` - (Required) Enter a unique Ocean cluster identifier. Cannot be updated.
+* `health` - (Optional) The Ocean AKS Health object.
+    * `grace_period` - (Optional, Default: `600`) The amount of time to wait, in seconds, from the moment the instance has launched until monitoring of its health checks begins.
+* `name` - (Required) Add a name for the Ocean cluster.
+* `scheduling` - (Optional) An object used to specify times when the cluster will turn off. Once the shutdown time will be over, the cluster will return to its previous state.
+    * `shutdown_hours` - (Optional) An object used to specify times that the nodes in the cluster will be taken down.
+        * `is_enabled` - (Optional) Flag to enable or disable the shutdown hours mechanism. When False, the mechanism is deactivated, and the cluster remains in its current state.
+        * `time_windows` - (Optional) The times that the shutdown hours will apply.
+* `headrooms` - (Optional) Specify the custom headroom per VNG. Provide a list of headroom objects.
+  * `cpu_per_unit` - (Optional) Configure the number of CPUs to allocate the headroom. CPUs are denoted in millicores, where 1000 millicores = 1 vCPU.
+  * `memory_per_unit` - (Optional) Amont of GPU to allocate for headroom unit.
+  * `gpu_per_unit` - (Optional) Configure the amount of memory (MiB) to allocate the headroom.
+  * `num_of_units` - (Optional) The number of units to retain as headroom, where each unit has the defined headroom CPU and memory.
+* `availability_zones` - (Optional) An Array holding Availability Zones, this configures the availability zones the Ocean may launch instances in per VNG.
+* `labels` - (Optional) An array of labels to add to the virtual node group.Only custom user labels are allowed, and not Kubernetes built-in labels or Spot internal labels.
+    * `key` - (Required) Set label key.
+    * The following are not allowed:
+    * "kubernetes.azure.com/agentpool"
```

Comment:
this shouldn't be bullet points

Reply:
removed bullet points and made it to list[string]